pkg/cloudprovider/providers/azure/azure_loadbalancer.go

```diff
@@ -948,6 +894,93 @@ func (az *Cloud) reconcileLoadBalancer(clusterName string, service *v1.Service,
 	return lb, nil
 }
 
+func (az *Cloud) createLoadBalancerRule(
+	expectedProbes *[]network.Probe,
+	expectedRules *[]network.LoadBalancingRule,
+	port v1.ServicePort,
+	service *v1.Service,
+	lbFrontendIPConfigID string,
+	lbBackendPoolID string,
+	lbName string,
+	lbIdleTimeout *int32) error {
+	lbRuleName := az.getLoadBalancerRuleName(service, port, subnet(service))
+
+	glog.V(2).Infof("createLoadBalancerRule lb name (%s) rule name (%s)", lbName, lbRuleName)
+
+	transportProto, _, probeProto, err := getProtocolsFromKubernetesProtocol(port.Protocol)
+	if err != nil {
+		return err
+	}
+
+	if serviceapi.NeedsHealthCheck(service) {
+		if port.Protocol == v1.ProtocolSCTP {
```

Comment:
`getProtocolsFromKubernetesProtocol()` has already checked this, so the check is not needed here